code/datums/mutantraces.dm

```diff
@@ -302,6 +304,49 @@
 		..()
 		return
 
+	proc/set_tail(var/mob/living/carbon/human/Tf as mob, var/tailnum as num)
+		if(!ishuman(Tf) || tailnum > ROACH_TAIL || tailnum < NO_TAIL)
+			return 0
+
+		if(Tf.organHolder)	// dump the old tail, if they have one
+			Tf.organHolder.tail = null
+
+		for(var/obj/item/organ/tail/hidden_secret_tails in Tf.contents)	// Cus mutants love to hoard tails in their mob
+			if(istype(hidden_secret_tails, /obj/item/organ/tail))
+				qdel(hidden_secret_tails)	// Please stop exploding into tails (thank you)
+
+		switch(tailnum)	// Mutantraces come with a free, new tail
+			if (NO_TAIL)	// technically you have a tail, its just invisible
+				Tf.organHolder.tail = new /obj/item/organ/tail/bone(Tf, Tf.organHolder)
+				Tf.organHolder.organ_list["tail"] = Tf.organHolder.tail
```

Comment:
why are you repeating this line every time

Reply:
321? Okay, I'll put that line after the switch is complete and remove the rest.